src/tcompiler.cpp

```diff
@@ -2993,16 +3045,38 @@ static int terra_linkllvmimpl(lua_State * L) {
     }
     #if LLVM_VERSION == 36
     ErrorOr<Module *> mm = parseBitcodeFile(mb.get()->getMemBufferRef(),*TT->ctx);
+    #elif LLVM_VERSION >= 60
+    Expected<std::unique_ptr<Module>> mm = parseBitcodeFile(mb.get()->getMemBufferRef(),*TT->ctx);
     #elif LLVM_VERSION >= 37
     ErrorOr<std::unique_ptr<Module>> mm = parseBitcodeFile(mb.get()->getMemBufferRef(),*TT->ctx);
     #else
     ErrorOr<Module *> mm = parseBitcodeFile(mb.get().get(),*TT->ctx);
     #endif
     if(!mm) {
         if(fromstring)
+            #if LLVM_VERSION >= 50
```

Comment:
Could we swap the `{` and the `#if`? Having the `if` statement be single-line in only one branch is making me nervous.

Reply:
Ahh. That is to have the minimal changes possible. Fixed :)